Runner.php

```diff
@@ -15,34 +16,28 @@ public function __construct($config, $server)
 
     public function queueDirectory($dir, $prefix = '')
     {
-        if(isset($this->config['include_paths'])) {
-            $this->queueWithIncludePaths();
-        } else {
-            $this->queuePaths($dir, $prefix, $this->config['exclude_paths']);
```

Comment:
Sorry, when `exclude_paths` is provided we're passed `["./"]` as the `include_paths` which means later in the code we try to read `/code/./` which makes the engine analyze all files regardless of extension/path.